scaffolder/publish-standalone-tech-docs/publish-standalone-tech-docs.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: publish-standalone-tech-docs
+  title: Publish a new standalone TechDocs repository and register with Roadie Backstage
+  description: "Publish a new repository with a catalog-info.yaml file with your company's standalone TechDocs details."
+spec:
+  owner: unspecified # TODO: Define who the owner of this template is. This is likely the team who own the relationship with Roadie. e.g. group:customer-name/platform
+  type: service
+
+  # These are the steps which are rendered in the frontend with the form input.
+  parameters:
+    - title: Repository Details # Title of the first step.
+      required:
+        - repoName
+      properties:
+        repoHost:
+          type: string
+          default: github.com
+          ui:widget: hidden
+
+        gitHubOrg:
+          type: string
+          default: roadie-demo # TODO: Set your GitHub Org value here.
+          ui:widget: hidden
+
+        repoName:
+          title: Repository Name
+          type: string
+          default: company-docs # TODO: Set your default name for the documentation repository here.
+
+        defaultBranch:
+          title: Repository Branch
+          type: string
+          default: main # TODO: Define your expected default branch.
+          enum: ['main']
+
+        repoVisibility:
+          title: Repository Visibility
+          type: string
+          default: private
+          enum: ['private', 'public']
+
+    - title: TechDocs Details # Title of the second step.
+      required:
+        - name
+        - description
+        - lifecycle
+      properties:
+        name:
+          title: Component Name
+          description: The name of the component in Roadie Backstage, typically the same name used in the repository. This will be used to create a human readable title.
+          type: string
+
+        description:
+          title: Component Description
+          decription: A Detailed decription allows for the component to be easily found and understood in Roadie Backstage.
+          type: string
+
+        lifecycle:
+          title: Component Lifecycle
+          type: string
+          default: production # TODO: Add your most common lifecycle here.
+          enum: ['production', 'experimental', 'development'] # TODO: What are the lifecycle environments you have?
+
+        owner:
+          title: Component Owner
+          type: string
+          description: Owner of the component
+          ui:field: OwnerPicker
+          ui:options:
+            catalogFilter:
+              kind: Group # TODO: Decide if you want to allow setting owner by user, group, or both. e.g. [Group, User]
+        
+        subDirectoryNames:
+          title: List of subdirectories
+          description: A List of TechDocs subdirectories to be created at repo publishing.
```

Comment:
I am unsure what this parameter does.. or why?

Reply:
This allows the user to add the names for all the subdirectories that will be created in the docs folder. There is logic in the Action step that takes this array and creates a folder with the value as the name and an `index.md` file for each.